### ISSUE TITLE
make test no use

### ISSUE THREAD

**Issue by xx789633** [yaledb/calvin#6]
Hi,
When I "make test", there are no executables generated. Could you possibily help me out? Many thanks.

**Comment by yaledb** [comment]
Yes,  there were no executables generated for "make test". Please follow Readme to run "make -j" to generate the executables.
